src/main/java/net/mcreator/ui/modgui/DimensionGUI.java

```diff
@@ -411,6 +415,7 @@ else if (page == 1)
 		portalSound.setSound(dimension.portalSound);
 		enableIgniter.setSelected(dimension.enableIgniter);
 		igniterName.setText(dimension.igniterName);
+		rarity.setSelectedItem(dimension.igniterRarity);

```

Comment:
Rename UI component to match igniterRarity too

Reply:
Done